source/simulator/assemblers/advection.cc

```diff
@@ -468,6 +468,307 @@ namespace aspect
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    template <int dim>
+    void
+    DarcySystem<dim>::create_additional_material_model_outputs(MaterialModel::MaterialModelOutputs<dim> &outputs) const
+    {
+      MeltHandler<dim>::create_material_model_outputs(outputs);
+    }
+
+    template <int dim>
+    void
+    DarcySystem<dim>::execute (internal::Assembly::Scratch::ScratchBase<dim>   &scratch_base,
+                                   internal::Assembly::CopyData::CopyDataBase<dim> &data_base) const
```

Comment:
The indentation looks off here, were you able to successfully run `make indent`?